src/stories/card.stories.js

```diff
@@ -35,18 +35,18 @@ export const Basic = () => html`
         of the card's content.
       </p>
       <cod-button
-        data-primary="true"
-        data-disable="false"
-        data-label="Link"
-        data-background-color="primary"
-        data-img=""
-        data-img-alt=""
-        data-icon=""
-        data-icon-order=""
-        data-icon-size=""
-        data-shape="fluid"
-        data-aria-label=""
-        data-link="https://detroitmi.gov"
+        primary="true"
+        disable="false"
+        label="Link"
+        background-color="primary"
+        img=""
+        img-alt=""
+        icon=""
+        icon-order=""
+        icon-size=""
```

Comment:
Also, there's a comment in #201 like this:

> Also, unless the attribute is a boolean attribute, the component should treat attribute="undefined" or attribute="" the same as omitting the attribute entirely. In Storybook stories, we should prefer omitting the attribute entirely if the storybook control is not set.

So for these non-boolean attributes where we're passing `""` or `"undefined"`, let's remove them entirely instead of continuing to pass empty values.

Note: We may need to also update `Button.js` to handle these attributes differently.